Cargo.toml

```diff
@@ -8,12 +8,13 @@ authors = [
 description = "Rust binding of NumPy C-API"
 documentation = "https://pyo3.github.io/rust-numpy/numpy"
 edition = "2018"
+rust-version = "1.48"
```

Comment:
This generates a warning on toolchains before `1.56`, so it might not be such a good idea to add it yet:

https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field

Reply:
Yes, it does but personally, I think machine readable information on MSRV is worth the nuisance. Especially since the warning will by default only be display when building the `rust-numpy` crate, not a crate which just depends on it, so it does not really affect user code.

Reply:
👍 that's a fair position, we can also make the argument that although we support compilers older than 1.56 we don't have to give older compilers maximum ergonomics. (E.g. in PyO3 compilers older than 1.52 don't get `FromPyObject` implementations for all `[T; N]`, as that requires const genetics.)

Let's keep this here then!